src/main/kotlin/entity/ChattORESpec.kt

```diff
@@ -45,6 +47,12 @@ object ChattORESpec : ConfigSpec("") {
         val joinDiscord by optional("**<player> has joined the network**")
         val leaveDiscord by optional("**<player> has left the network**")
     }
+    val nicknamePresets by optional(
+        mapOf(
+            "pride" to prideColors.toGradientMinimessage(),
```

Comment:
Where is `toGradientMinimessage()` defined? I don't see it anywhere.

Reply:
declared a function in config seems like i added it with my 2n last commit but deleted it again with my last commit 2 months ago lmao